downstream/modules/platform/con-aap-upgrade-planning.adoc

```diff
@@ -28,3 +28,9 @@ Before you begin the upgrade process, review the following considerations to pla
 [role="_additional-resources"]
 .Additional resources
 * <<editing-inventory-file-for-updates_{context}, Setting up the inventory file >>
+
+[discrete]
+== {EDAcontroller}
+//ATTENTION: Remove this section for AAP 2.5; customers will no longer need to perform deactivation because services will be automatically restored after upgrade and migration. 
```

Comment:
```suggestion
//ATTENTION: Remove this section for EDA 1.0.4; customers will no longer need to perform deactivation because services will be automatically restored after upgrade and migration. 
```

Reply:
Okay. Duly noted. When will EDA 1.0.4 be released? Will that happen with AAP 2.5 or will it be done with separately?

Reply:
Thanks for reviewing the update. The note was added for me or anyone else on the docs team. It won't be viewed by customers.

Reply:
1.0.4 will be released in 2.4, thats why I think its better to point to eda 1.0.4 instead of aap 2.5

Reply:
Got it! Thanks for the clarification.